misc/python/materialize/zippy/balancerd_actions.py

```diff
@@ -0,0 +1,62 @@
+# Copyright Materialize, Inc. and contributors. All rights reserved.
+#
+# Use of this software is governed by the Business Source License
+# included in the LICENSE file at the root of this repository.
+#
+# As of the Change Date specified in that file, in accordance with
+# the Business Source License, use of this software will be governed
+# by the Apache License, Version 2.0.
+
+
+from materialize.mzcompose.composition import Composition
+from materialize.zippy.balancerd_capabilities import BalancerdIsRunning
+from materialize.zippy.framework import Action, Capability
+from materialize.zippy.mz_capabilities import MzIsRunning
+
+
+class BalancerdStart(Action):
+    """Starts balancerd"""
+
+    @classmethod
+    def requires(cls) -> set[type[Capability]]:
+        return {MzIsRunning}
+
+    @classmethod
+    def dislikes(cls) -> set[type[Capability]]:
```

Comment:
`dislikes` sounds like a preference and not a strong restriction. Is this the case?
Alternatively, consider renaming it to `forbids` / `excludes` / `incompatible_with`?

Reply:
incompatible_with it is!